packages/esm-appointments-app/src/home-appointments/actionable-notifications/notification.component.scss

```diff
@@ -0,0 +1,6 @@
+.actionableNotification {
+  z-index: 100000;
+  position: fixed;
+  width: 100%;
+  bottom: 2rem;
```

Comment:
spacing-07

Reply:
oh sorry I forgot about the spacing variables, I will fix that.